From: Lennart Weller <lhw@ring0.de>
Date: Tue, 18 Dec 2018 12:19:11 +0100
Subject: Makefile changes for debian packaging

 - Change build_version to debian package version for easier debugging
 - Include debian package buildflags
---
 client/CMakeLists.txt | 11 ++---------
 1 file changed, 2 insertions(+), 9 deletions(-)

diff --git a/client/CMakeLists.txt b/client/CMakeLists.txt
index e83aaf9..651739a 100644
--- a/client/CMakeLists.txt
+++ b/client/CMakeLists.txt
@@ -3,7 +3,7 @@ project(looking-glass-client C)
 
 set(CMAKE_MODULE_PATH "${PROJECT_SOURCE_DIR}/cmake/")
 
-SET(CMAKE_C_FLAGS "-std=gnu99 -g -O3 -march=native -Wall -Werror -Wfatal-errors -ffast-math -fdata-sections -ffunction-sections")
+SET(CMAKE_C_FLAGS "$ENV{CFLAGS} $ENV{CPPFLAGS} -std=gnu99 -ffast-math -fdata-sections -ffunction-sections")
 SET(CMAKE_EXE_LINKER_FLAGS "-Wl,--gc-sections")
 
 find_package(PkgConfig)
@@ -21,16 +21,9 @@ pkg_check_modules(PKGCONFIG REQUIRED
 	hogweed
 )
 
-execute_process(
-	COMMAND			git describe --always --long --dirty --abbrev=10 --tags
-	WORKING_DIRECTORY	${PROJECT_SOURCE_DIR}
-	OUTPUT_VARIABLE		BUILD_VERSION
-	OUTPUT_STRIP_TRAILING_WHITESPACE
-)
-
 find_package(GMP)
 
-add_definitions(-D BUILD_VERSION='"${BUILD_VERSION}"')
+add_definitions(-D BUILD_VERSION='"debian/$ENV{DEB_VERSION}"')
 add_definitions(-D USE_NETTLE)
 add_definitions(-D ATOMIC_LOCKING)
 add_definitions(-D GL_GLEXT_PROTOTYPES)
